File: domain/src/main/java/com/github/anrimian/musicplayer/domain/models/equalizer/Band.java
package com.github.anrimian.musicplayer.domain.models.equalizer;

public class  Band {

    private final short bandNumber;
    private final int[] frequencyRange;
    private final int centerFreq;

    public Band(short bandNumber,
                int[] frequencyRange,
                int centerFreq) {
        this.bandNumber = bandNumber;
        this.frequencyRange = frequencyRange;
        this.centerFreq = centerFreq;
    }

    public int getCenterFreq() {
        return centerFreq;
    }

    public short getBandNumber() {
        return bandNumber;
    }

    public int[] getFrequencyRange() {
        return frequencyRange;
    }
}

File: domain/src/main/java/com/github/anrimian/musicplayer/domain/models/equalizer/Band.kt
package com.github.anrimian.musicplayer.domain.models.equalizer;

public class  Band {

    private final short bandNumber;
    private final int centerFreq;

    public Band(short bandNumber, int centerFreq) {
        this.bandNumber = bandNumber;
        this.centerFreq = centerFreq;
    }

    public int getCenterFreq() {
        return centerFreq;
    }

    public short getBandNumber() {
        return bandNumber;
    }

}
